go.mod

```diff
@@ -9,27 +9,27 @@ require (
 	github.com/foxcpp/go-mockdns v0.0.0-20210729171921-fb145fc6f897
 	github.com/ghodss/yaml v1.0.0
 	github.com/gonum/graph v0.0.0-20190426092945-678096d81a4b
-	github.com/google/go-cmp v0.5.9
+	github.com/google/go-cmp v0.6.0
 	github.com/imdario/mergo v0.3.8
 	github.com/miekg/dns v1.1.25
-	github.com/openshift/api v0.0.0-20231129134630-a782d1c1541c
+	github.com/openshift/api v0.0.0-20231218131639-7a5aa77cc72d
 	github.com/openshift/build-machinery-go v0.0.0-20230228230858-4cd708338479
-	github.com/openshift/client-go v0.0.0-20230926161409-848405da69e1
-	github.com/openshift/library-go v0.0.0-20231130112926-2285c05489d7
+	github.com/openshift/client-go v0.0.0-20231218140158-47f6d749b9d9
+	github.com/openshift/library-go v0.0.0-20240305144041-18ee8279b4e3
```

Comment:
/hold

this is bringing 4.16 library-go into 4.15 kube-apiserver-operator. Unless or until we need to update client-go from an unsupported level, this should remain in sync

Reply:
My fault, https://github.com/openshift/library-go/pull/1688 is 4.15 cherrypick for library-go

Reply:
Updated